Makefile

```
@@ -66,6 +71,14 @@ clean-db:  ## remove LMS databases
 	$(COMPOSE) rm edx_mongodb edx_mysql edx_redis keycloak_postgres
 .PHONY: clean-db
 
+demo-course: \
```

As there will be multiple LMSs in the playground, what about prefixing rules by the LMS name?

```suggestion
edx-demo-course: \
```